### DISCUSSION TITLE
希望可以增加个连接后执行命令功能

### DISCUSSION THREAD

**Discussion by YoYulyvW** [lollipopkit/flutter_server_box#67] [Ideas]
有些服务器要sudo -i获取权限，每次连接都要手动输入一次好麻烦哟

**Comment by lollipopkit** [comment]
指的是snippet吗？

现在已经有了，先添加snippet，再点击ssh终端虚拟键盘右上角的按钮选择

**Reply by YoYulyvW** [reply]
不是这个哦，是连接ssh后自动执行的命令像sudo -i 跟密码

**Reply by MacKenia** [reply]
也许可以自行修改 `.config/server_box/mobile_v34.sh` 实现

**Reply by YoYulyvW** [reply]
> 也许可以自行修改 `.config/server_box/mobile_v34.sh` 实现

为啥我在安装目录 数据目录都没找到这个路径

**Comment by MacKenia** [comment]
实测我的每台 Linux Server 都会在 `.config/server_box/` 目录下出现 `mobile_vxx.sh` 文件，有可能你找错用户了，例如你ServerBox指定的 `root` 用户，而 ssh 登录服务器却是另一个用户，可能就会找不到

**Reply by YoYulyvW** [reply]
> 实测我的每台 Linux Server 都会在 `.config/server_box/` 目录下出现 `mobile_vxx.sh` 文件，有可能你找错用户了，例如你ServerBox指定的 `root` 用户，而 ssh 登录服务器却是另一个用户，可能就会找不到

收到，明白了，我还以为是app本身的路径

**Comment by lollipopkit** [comment]
已在764中实现，需编辑代码片段，设置自动运行
